packages/terra-application/README.md

```diff
@@ -7,20 +7,13 @@
 The application component sets global styles for the entire application. Global styles include font, colors, margins, and sizes.
 
 - [Getting Started](#getting-started)
-- [Documentation](#documentation)
+- [Documentation](docs/)
```

Comment:
doc urls should look like `https://github.com/cerner/terra-ui/tree/master/packages/terra-application/docs`